prdoc/pr_3761.prdoc

```diff
@@ -0,0 +1,25 @@
+title: "Snowbridge: Synchronize from Snowfork repository"
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      This PR improves the beacon client to send the execution header along with the message as proof and removes the verification and storing of all execution headers.
+      If the AH sovereign account is depleted and relayer rewards cannot be paid, the message should still be processed.
```

Comment:
is this still the case?

Reply:
It is not, removed in https://github.com/paritytech/polkadot-sdk/pull/4829/commits/1877257da7bd8f0e92ad0e8abdbf64ff91865173. :) Good catch